src/components/pearl-chain/pearl-chain.ts

```diff
@@ -58,7 +62,11 @@ export class SbbPearlChainElement extends LitElement {
   }
 
   private _isAllCancelled(legs: PtRideLeg[]): boolean {
```

Comment:
probably intended but sound kind of wrong that "all" canceled means that just some leg is cancelled or only even partially cancelled?

Reply:
Yes, it's intended. It should appear as if it's completely canceled, including the partially canceled ones and those with canceled legs